lib/IDE/CodeCompletion.cpp

```diff
@@ -1130,6 +1130,8 @@ static void addPoundDirectives(CodeCompletionResultSink &Sink) {
     Builder.addSimpleTypedParameter("Int");
     Builder.addRightParen();
   });
+
+#ifndef SWIFT_SWIFT_PARSER
```

Comment:
I had originally removed this, but better to keep it for linux/windows for now. We can remove after SWIFT_SWIFT_PARSER is enabled in all the places.